tests/integration/sync_test.go

```diff
@@ -0,0 +1,116 @@
+//go:build integration
+
+package e2e
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"reflect"
+	"sort"
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+var DECK = filepath.Join("../../deck")
```

Comment:
This will result in a bad developer experience - a compile step is required whenever a test has to run.
Can you please instead invoke main() from here? You can use https://pkg.go.dev/testing#B.Setenv to set the right environment variables for configuration.